Python/pywarpx/picmi.py

```diff
@@ -1335,6 +1335,7 @@ def initialize_inputs(self):
             pywarpx.diagnostics._diagnostics_dict[self.name] = self.diagnostic
 
         self.diagnostic.diag_type = 'Full'
+        self.diagnostic.fields_to_plot = 'none'
```

Comment:
Thanks for posting this PR. This new line should go after line 1334 that creates a new `Diagnostic` instance. If a previously created `Diagnostic` instance is fetched (in line 1332) it may have be made by a `FieldDiagnostic` and so already have `fields_to_plot` defined.